### PR TITLE
Feature/analytics code

### PR DESCRIPTION
**Pull request by 55**
Alternative approach to #2288

WIP

### CODE REVIEW

File: core/client/analytics.js [hunk 1]
```diff
@@ -0,0 +1,12 @@
+/* Copyright 2017 Apinf Oy
+This file is covered by the EUPL license.
+You may obtain a copy of the licence at
+https://joinup.ec.europa.eu/community/eupl/og_page/european-union-public-licence-eupl-v11 */
+
+import OKGAnalytics from '@okgrow/auto-analytics';
```
Comment (mauriciovieira) [inline core/client/analytics.js]: You can use an alternative [import](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/import) here.

`import { OKGAnalytics as okGAnalytics } from '@okgrow/auto-analytics';`

File: core/client/analytics.js [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+
+const analytics = {
+  'Google Analytics': { trackingId: '' },
+};
+// eslint-disable-next-line
```
Comment (mauriciovieira) [inline core/client/analytics.js]: then remove this eslint-disable

File: core/client/analytics.js [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+OKGAnalytics(analytics);
```
Comment (mauriciovieira) [inline core/client/analytics.js]: and

`okGAnalytics(analytics);`

**Comment by mauriciovieira** [comment]
Also, please run `yarn` and update yarn.lock

File: core/client/analytics.js [hunk 4]
```diff
@@ -3,10 +3,25 @@ This file is covered by the EUPL license.
 You may obtain a copy of the licence at
 https://joinup.ec.europa.eu/community/eupl/og_page/european-union-public-licence-eupl-v11 */
 
-import OKGAnalytics from '@okgrow/auto-analytics';
```
Comment (mauriciovieira) [inline core/client/analytics.js]: Don't forget to add our standardized import comments.

File: core/client/analytics.js [hunk 5]
```diff
@@ -3,10 +3,25 @@ This file is covered by the EUPL license.
...
+import { Meteor } from 'meteor/meteor';
 
-const analytics = {
-  'Google Analytics': { trackingId: '' },
-};
-// eslint-disable-next-line
-OKGAnalytics(analytics);
+import { OKGAnalytics as okGAnalytics } from '@okgrow/auto-analytics';
```
Comment (mauriciovieira) [inline core/client/analytics.js]: Please see our [importing](https://github.com/apinf/platform/blob/develop/CONTRIBUTING.md#importing) comments.

**Comment by 55** [comment]
Temporarily blocked. Opened an upstream [issue](https://github.com/okgrow/auto-analytics/issues/4).

**Comment by 55** [comment]
@marla-singer if you have time, could you help with this PR? 
I have created an upstream issue explaining what we are trying to achieve: https://github.com/okgrow/auto-analytics/issues/4#issue-234778783

**Comment by marla-singer** [comment]
@55 What kind of help would you like from me?

**Comment by 55** [comment]
I am trying to update `settings` object reactively. E.g.: https://github.com/apinf/platform/blob/6beaefb18dd34ec495d35a1c7634151d834ae4a8/core/client/analytics.js#L13-L26
```js
  // Get branding
  const branding = Branding.findOne();

  if (branding) {
    const googleAnalytics = branding.analytics.googleAnalytics;

    if (googleAnalytics) {
      const settings = {
        'Google Analytics': { trackingId: googleAnalytics },
      };
      // Run OKGrow analytics
      okGAnalytics(settings);
    }
  }
```

It might take whatever value from settings.json or directly from `const settings = { 'Google Analytics': { trackingId: 'test value' }};` only once and `okGAnalytics(settings);` never runs a second time. 

I am trying to find a way to update the `settings` object reactively, in our case from brading page.

**Comment by marla-singer** [comment]
@55  Unfortunately, I've not had an experience with "settings" parameter in Meteor and don't have any idea how to resolve it

**Comment by mauriciovieira** [comment]
Since this PR is supposed to fix #2288, and #2288 is fixed, should we just abort this trench?

**Pull request closed without merging**